src/specVersions/v0.2/RicardianContractProcessorImpl.test.ts

```diff
@@ -0,0 +1,88 @@
+import { Abi } from '../../interfaces'
+import { RicardianContractProcessorImpl } from './RicardianContractProcessorImpl'
+
+import { eosioTokenAbi, transferTransaction } from '../../testfixtures/fixtures'
+
+const transaction = JSON.parse(transferTransaction)
+const abi = JSON.parse(eosioTokenAbi)
+
+// tslint:disable:max-line-length
+describe('RicardianContractProcessorImp - v0.2', () => {
+  const rcp = new RicardianContractProcessorImpl()
+  // const contractMetadata = '---\ntitle: Token Transfer\nsummary: Transfer tokens between accounts.\nicon: https://a.com/token-transfer.png#00506E08A55BCF269FE67F202BBC08CFF55F9E3C7CD4459ECB90205BF3C3B562\n---\n\n'
+
+  it('dumps variable as JSON', () => {
+    // tslint:disable:max-line-length
+    const expectedHtml =
```

Comment:
blanket comment to the template literals in this file, could they be indented in properly? makes it a bit harder to read down the file

Reply:
I agree. But the difficulty, especially with the `expected` values, is you're at the mercy of what is ultimately generated by markdown. If that gets indented, then it doesn't match what gets output. Could split it up into individual concatenated strings, but that's just a different and (to me at least) even harder to maintain form.

I kind of feel like there should be a better way to structure all these tests in general to make them more readable. Tho anything where you have to deal with big blobs of text like that can be painful!

Reply:
Ahh gotcha, bummer. Well if that's the case, I think what you have now might be the best solution i can think of atm